package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@eigenspace/helper-scripts",
     "version": "1.0.73",
-    "main": "index.js",
+    "main": "src/index.ts",
```

Comment:
We also should not forget about built package where there are not typescript files.